### PR TITLE
Add shared-components coverage

### PR DESCRIPTION
**Pull request by adodge-bcg**
# Description

This PR includes the following proposed change(s):

Add shared-components coverage to the summed coverage report

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Tested with git action

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/code-climate-coverage-aggregation.yml [hunk 1]
```diff
@@ -136,6 +136,11 @@ jobs:
               fi
           done
 
+      # Formatting the SHARED COMPONENTS coverage report
+      - name: Format the SHARED COMPONENTS coverage report
+        run: |
```
Comment (akroon3r) [inline .github/workflows/code-climate-coverage-aggregation.yml:141]: you only need the | if you're doing multi line bash commands, but potato potatoe.

**Comment by akroon3r** [comment]
@adodge-bcg maybe just close this PR, and open a new branch against master with the only change you need.

also congrats.

**Comment by sdevalapurkar-bcgov** [comment]
we could just squash the commits :)

**Comment by akroon3r** [comment]
more about undoing commits to get the required behaviour than cleaning commit history

**Comment by sdevalapurkar-bcgov** [comment]
@adodge-bcg  - can we pls add cc exclusions for anything not inside `src/blah/*.js`?

**Comment by adodge-bcg** [comment]
Up to you guys about the new vs squash, I can do ether.
Updated the codeclimate ignore with a more generic parter for frontend files, let me know if that's what you were thinking @sdevalapurkar

**Comment by sdevalapurkar-bcgov** [comment]
thanks alan

**Comment by sdevalapurkar-bcgov** [comment]
still looks like it is looking at test coverage for files such as assetstransformer... what you did seems correct though

File: src/frontend/shared-components/jest.config.js [hunk 2]
```diff
@@ -20,6 +20,6 @@ module.exports = {
   verbose: true,
   testResultsProcessor: "jest-sonar-reporter",
   collectCoverage: false,
-  coverageReporters: ["text", "lcov"],
+  coverageReporters: ["text", ["lcov", {"projectRoot": "../../../"}]],
```
Comment (adodge-bcg) [inline src/frontend/shared-components/jest.config.js:22]: Fix from https://github.com/facebook/jest/issues/9773, based on a comment from jcdesousa

Comment (akroon3r) [reply src/frontend/shared-components/jest.config.js:22]: solid reference.

**Pull request merged**